### PR TITLE
1.27+28 release notes

### PR DESCRIPTION
**Pull request by PtitBen**
Added latest release notesAdding 1.27/1.28 release notes

### CODE REVIEW

File: docs/community/releases.md [hunk 1]
```diff
@@ -4,9 +4,106 @@ description: Here are our release notes - enjoy!
...
+
+Release date - 2021/03/16
+
+**New features 🔘**
+
+* `ns-selector` can now be decorated with a keyline to colour-code or highlight elements!
+
+
+**Improvements 🏵**
+* Decorations can no be used with Nucleus Experiences (`nsx-`).
```
Comment (PraveenAsokan) [inline docs/community/releases.md:72]: typo - no used instead of now

File: docs/community/releases.md [hunk 2]
```diff
@@ -4,9 +4,106 @@ description: Here are our release notes - enjoy!
...
+
+* `ns-password`: a brand new password component is now available, and it's called `ns-password`! This is the first of two releases and allows to capture an existing password. Stay tuned for the password creation side of things! (the documentation for the component will be released with the part)
+
+
+**Improvements 🖋**
+* `ns-content` can now be consumed by *any* (most) component(s)
+
+**Multibrand enablement and design tokens 🪙**
+
+* `ns-testimonial` - typography tokens added
```
Comment (PraveenAsokan) [inline docs/community/releases.md:25]: Can we use either ```typography token added``` or ```Added typography token``` (which one is apt) for all components mentioned above?

File: docs/community/releases.md (hunk 1 above)
Comment (PtitBen) [reply docs/community/releases.md:72]: Good spot!

File: docs/community/releases.md (hunk 2 above)
Comment (PtitBen) [reply docs/community/releases.md:25]: Fixed the first one so it matches the rest!

File: docs/community/releases.md [hunk 3]
```diff
@@ -4,9 +4,106 @@ description: Here are our release notes - enjoy!
...
-## Version 1.26.0
+**Tokens, tokens, tolkens 🪙🪙🪙 (oh, and passw ᐧ ᐧ ᐧ ᐧ!)**
+
+Last release, we set the groundwork to start lifting the brand layer off of our components, and store those styles in tokens. Well, in this release, things are definitely in motion. We also have released the first half of a brand new password component.
+
+Release date - 2021/04/08
+
+**New features ᐧ ᐧ ᐧ ᐧ**
+
+* `ns-password`: a brand new password component is now available, and it's called `ns-password`! This is the first of two releases and allows to capture an existing password. Stay tuned for the password creation side of things! (the documentation for the component will be released with the part)
```
Comment (RobTobias123) [inline docs/community/releases.md:17]: Suggest change to ... This is the first of two release and allows **the** capture of an existing password.

Comment (RobTobias123) [reply docs/community/releases.md:17]: Also... capitalise the initial letter in the brackets and perhaps reword to... (The documentation will be released with that part of the component).

**Review by PraveenAsokan — APPROVED** [review]

File: docs/community/releases.md [hunk 4]
```diff
@@ -4,9 +4,106 @@ description: Here are our release notes - enjoy!
...
+* `ns-download` - Added typography tokens
+* `ns-footer` - Added typography tokens
+* `ns-editorial` - Added typography tokens
+* `ns-accordion` - Added typography tokens
+
+
+
+**Bug fixes 🐛**
+
+* `ns-lockup` - Over-ride font size fix
```
Comment (RobTobias123) [inline docs/community/releases.md:44]: Override is generally not hyphenated?

File: docs/community/releases.md [hunk 5]
```diff
@@ -4,9 +4,106 @@ description: Here are our release notes - enjoy!
...
+* `ns-datepicker` - the calendar clashed with it's surroundings in some scenarios. This is now fixed (first solution is content is pushed to allow calendar to open, a second more refined improvement is coming)
```
Comment (RobTobias123) [inline docs/community/releases.md:45]: **its** not **it's** in this context

File: docs/community/releases.md (hunk 4 above)
Comment (PtitBen) [reply docs/community/releases.md:44]: I hesitated ... 🤦 I'll fix it now!

File: docs/community/releases.md (hunk 5 above)
Comment (PtitBen) [reply docs/community/releases.md:45]: 😱 I can't believe I made one of those!  Thanks !!

File: docs/community/releases.md (hunk 3 above)
Comment (PtitBen) [reply docs/community/releases.md]: changed both!

**Pull request merged**